package.json

```diff
@@ -7,7 +7,7 @@
     "cross-env": "dist/bin/cross-env.js"
   },
   "engines": {
-    "node": ">=4.0"
+    "node": ">=4.8"
```

Comment:
Let's add a note somewhere in the README indicating Node Version support. I expect some people who have been using `cross-env` today will need to upgrade node before they can continue using it after this change.

How about we use this badge?

[![node-version][node-version-badge]][node]

```
[![node-version][node-version-badge]][node]

<!-- down below, with the rest of the links -->
[node-version-badge]: https://img.shields.io/badge/node-%3E%3D%204.8-orange.svg?style=flat-square
```

Put it after the `npm` version badge and before the downloads badge. What do you think?

[node-version-badge]: https://img.shields.io/badge/node-%3E%3D%204.8-orange.svg?style=flat-square
[node]: https://nodejs.org